LPS-106680 SF StrinBundler.concat

csierra opened the pull request.

Commit:
LPS-105158 New ScopeDescriptor based on scope alias and companyId

Commit:
LPS-105158 New ScopeDescriptor based on scope alias and companyId

Commit:
LPS-105158 New taglib to render a tree view

Commit:
LPS-105158 New util class to generate a tree based on a Set (using JGraphT), and test class for it

Commit:
LPS-105158 New DisplayContext to render the scopes list as a tree using the new taglib and utilities

Commit:
LPS-105158 Show new tree view based on OmniAdmin Permissions

Commit:
LPS-105158 BuildLang

Commit:
LPS-105158 Use Inmutable API

Commit:
LPS-105158 Refactor test

Commit:
LPS-105158 Sort scopes tree

Commit:
LPS-105158 SF rename to match tracked class

Commit:
LPS-105158 close service tracker map

Commit:
LPS-105158 SF sort

constants come first

Commit:
LPS-105158 SF simplify

Commit:
LPS-105158 SF rename

Commit:
LPS-105158 SF not needed

Commit:
LPS-105158 make dynamic

Commit:
LPS-105158 SF rename

Commit:
LPS-105158 SF rename

Commit:
LPS-105158 SF rename

Commit:
LPS-105158 SF simplify

Commit:
LPS-105158 SF simplify

Commit:
LPS-105158 SF simplify

Commit:
LPS-105158 SF rename consistently

Commit:
LPS-105158 No needed as a field

Commit:
LPS-105158 Make immutable

Commit:
LPS-105158 SF rename consistently

Commit:
LPS-105158 SF rename

Commit:
LPS-105158 SF rename

Commit:
LPS-105158 Generalize

Commit:
LPS-106680 SF StrinBundler.concat

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

Commit:
LPS-106680 SF Make private

Commit:
LPS-106680 SF Rename

Commit:
LPS-106680 Change to direct invocation to render

Commit:
LPS-106680 Use direct style render invocation

Commit:
LPS-106680 Correct parent handling

Commit:
LPS-106680 New tag with separate concerns

so one tag resets the state and the other colaborates so it is clear when the tags can be nested.
